core/cli/src/informant.rs

```diff
@@ -20,8 +20,8 @@ use client::{backend::Backend, BlockchainEvents};
 use futures::{Future, Stream};
 use futures03::{StreamExt as _, TryStreamExt as _};
 use log::{info, warn};
-use runtime_primitives::generic::BlockId;
-use runtime_primitives::traits::Header;
+use sr_primitives::generic::BlockId;
```

Comment:
Same

Reply:
For example about the above argument, ^^ should clearly be a one-liner.